scripts/album.js

```diff
@@ -190,6 +190,50 @@ album.add = function() {
 
 };
 
+album.addandmove = function (photoIDs) {
+
+	const action = function(data) {
+
+		let title = data.title;
+
+		const isNumber = (n) => (!isNaN(parseFloat(n)) && isFinite(n));
```

Comment:
I think it should be parseInt() instead of parseFloat.

Reply:
Possibly. That's taken directly from album.add (Line 156). Would you like me to change both?

Reply:
please do with `parseInt(x, 10)` (specify the base used in order to make sure that the use over different platform is the same). :)

Reply:
Done :)